samples/scala-reliable-timers/src/main/scala/com/example/actions/OrderAction.scala

```diff
@@ -0,0 +1,122 @@
+package com.example.actions
+
+import java.util.UUID
+
+import com.example.domain.Order
+import io.grpc.Status
+import io.grpc.StatusRuntimeException
+import org.slf4j.LoggerFactory
+
+// FIXME: we need to provide a default dispatcher for users to use
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Scala users deserve something better for this.

Reply:
Why, what is wrong with global (or one the user defined/decided on)?

Usually in Akka applications you should use the Akka dispatchers so that futures triggered by actors etc isn't staying around if you terminate the actor system, but for a Kalix app that is less important as the lifecycle is more tied to the JVM which will be killed on stop.

Reply:
In our examples, we are just transforming data (map/flatMap), so global might be fine, but are we ok in using it for calling external services? Or should we just tell users to defined their own execution context if they do more then map/flatMap?

Reply:
`global` is just an ordinary ForkJoinPool so that should be fine. Exposing Akka's default execution context could be slightly more efficient because one less thread pool, but maybe not worth it if it becomes additional API that the user has to learn. If we could put Akka's execution context in scope by default that might be nice and convenient.

Reply:
What about blocking calls? 

My initial idea was to expose Akka's default and blocking through the action context.